packages/errors/src/messages.ts

```diff
@@ -84,6 +85,7 @@ export const SolanaErrorMessages: Readonly<{
         'Invalid data enum variant. Expected one of [$variants], got $value',
     [SOLANA_ERROR__CODECS_INVALID_SCALAR_ENUM_VARIANT]:
         'Invalid scalar enum variant. Expected one of [$variants] or a number between $minRange and $maxRange, got $value',
+    [SOLANA_ERROR__CODECS_INVALID_STRING_FOR_BASE]: 'Expected a string of base $expectedBase, got $value.',
```

Comment:
This is totally fine, but the root cause of the error is _actually_ more like ‘this string contains a character that's not in this alphabet.’ You could _theoretically_ have a ‘base 58’ number that's based on a different alphabet than `123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz`.

I think it's more to the point to be like ‘hey, this string has characters that aren't in this alphabet; allowable characters are \`${alphabet}\`’.